test/gen_cat_test.html

```diff
@@ -0,0 +1,113 @@
+<html>
+<head>
+	<title>SPOT investigation: autogenerating inputs</title>
+
+	<link rel="stylesheet" type="text/css" href="../spot.css">
+
+	<script src="../lib/jszip.min.js"></script>
+    <script src="../trees/abstractMatchTrees.js"></script>
+	
+
+	<!-- to keep spot.js in sync with the codebase make sure to run jsbuild.sh after making any changes to javascript files in the main directory or in the constraints sub-directory
+	(open a terminal, cd into the main directory, type ./jsbuild.sh and hit enter) -->
+	<script src="../build/spot.js"></script>	
+
+				
+</head>
+
+<body style="padding-left: 5%; padding-right: 5%; padding-top: 20px">
+	<h2>GENerating inputs</h2>
+	<pre id="results-container"></pre>
+
+<script>
+	// ---- Variables that need to be customized ----
+	
+	/*Constraint sets.*/
+	var MatchSPMatchPS = ['binMaxBranches-phi', 'binMinBranches-phi', 'matchSP-xp'];
+	
+	/*A list of names for constraint sets.*/
+    var conNames = ['MatchSPMatchPS'];
+	
+	//Candidate sets are generated with the options below.
+     
+    var i_phi_w = {rootCategory:'i', recursiveCategory:'phi', terminalCategory: 'w'};
```

Comment:
Is this file supposed to be added by this pull request? Looks like a leftover from manual testing for a different feature.

Reply:
Moved this file to test/manual_testing, and updated it slightly.